vnext/Chakra/ChakraHelpers.cpp

```diff
@@ -122,7 +122,7 @@ class ChakraVersionInfo {
 
 class BytecodePrefix {
  public:
-  static std::pair<bool, BytecodePrefix> getBytecodePrefix(JsValueRef scriptFileName, uint64_t bundleVersion) noexcept {
```

Comment:
>scriptFileName [](start = 70, length = 14)

This param is unused. Not fully related to the change but it was kind of confusing when I saw it so I added it to this.